internal/lib/checkpoint.go

```diff
@@ -54,15 +54,15 @@ func (c *ContainerServer) ContainerCheckpoint(ctx context.Context, opts *Contain
 		return "", fmt.Errorf("failed to checkpoint container %s: %w", ctr.ID(), err)
 	}
 	if opts.TargetFile != "" {
-		if err := c.exportCheckpoint(ctr, specgen.Config, opts.TargetFile); err != nil {
+		if err := c.exportCheckpoint(ctx, ctr, specgen.Config, opts.TargetFile); err != nil {
 			return "", fmt.Errorf("failed to write file system changes of container %s: %w", ctr.ID(), err)
 		}
 	}
-	if err := c.storageRuntimeServer.StopContainer(ctr.ID()); err != nil {
+	if err := c.storageRuntimeServer.StopContainer(ctx, ctr.ID()); err != nil {
 		return "", fmt.Errorf("failed to unmount container %s: %w", ctr.ID(), err)
 	}
 	if err := c.ContainerStateToDisk(ctx, ctr); err != nil {
-		logrus.Warnf("Unable to write containers %s state to disk: %v", ctr.ID(), err)
+		logrus.WithContext(ctx).Warnf("Unable to write containers %s state to disk: %v", ctr.ID(), err)
```

Comment:
since we have ctx we can replace this with the internal log package instead of logrus

Reply:
I am open either way, but I agree we should do a `port all logrus` commit

Reply:
if we're going to change and then immediately change again, I guess doing it here makes sense

Reply:
good point, added a new commit to port these